### ISSUE TITLE
Cell metadata saving keyboard shortcut.

### ISSUE THREAD

**Issue by afshin** [jupyterlab/jupyterlab#6163]
**Describe the bug**
In the cell metadata editor the `Accel S` shortcut saves the notebook file saves, but the actual cell metadata doesn't save. Cell metadata seems to only save with a click on the checkbox button.

**To Reproduce**
1. Open a notebook.
1. Open notebook tools sidebar pane.
1. Modify "Cell Metadata"
1. Hit `Accel S`

**Expected behavior**
It think it should probably save the cell metadata when the save keyboard shortcut is invoked.

**Version**
Current `master` as of April 4.

**Comment by jasongrout** [comment]
Should we also save the notebook file too? That could be confusing.

**Comment by afshin** [comment]
This should probably be implemented by creating a command that first checks the metadata editor to make sure the content being saved is valid JSON ... and if it succeeds calling the `docmanager:save` command, otherwise showing an error dialog informing the user that saving the file was not possible.

**Comment by afshin** [comment]
cc: @tgeorgeux might have some ideas about what should go in the aforementioned error dialog.

**Comment by DancingQuanta** [comment]
Hi, any progress on this?

**Comment by marisaaquilina** [comment]
@DancingQuanta Planning to work on this issue!

**Comment by isabela-pf** [comment]
Here's two options for how users could be notified that invalid JSON will prevent the cell metadata from saving: 

1. The dialog seems to be standard for most of the errors in JupyterLab, notebook and more.
![Dialog 1](https://user-images.githubusercontent.com/50221806/80044260-c9a36900-84b8-11ea-8bd2-5f9796585660.png)

2. Follow the more subtle pattern of JSON syntax error message. Currently, I think this approach might be unique to the Settings text editor.
![Header Note 2](https://user-images.githubusercontent.com/50221806/80044277-d4f69480-84b8-11ea-8647-f29786af7762.png)
(For reference, this is where I'm drawing inspiration from.)
![Screen Shot 2020-04-22 at 3 48 22 PM](https://user-images.githubusercontent.com/50221806/80044291-e770ce00-84b8-11ea-80d2-1a92bd5d9918.png)

Basically this is a question of how severe a failure to save cell metadata might be to users. A dialog will ensure it is seen, but be a more disruptive experience. The small pop up message can be more easily overlooked, but can happen right by the cell metadata editor. 

I'd love feedback for which people think would be most appropriate since how people use cell metadata is not something I've had the opportunity to look into.
